webdriver/tests/element_click/element_click.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click
+def test_scroll_into_view(session):
+    session.url = inline("""
+        <input type=text value=Federer 
+        style="position: absolute; left: 0vh; top: 500vh">""")
+
+    element = session.find.css("input", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    # Check if element clicked is scrolled into view
+    assert session.execute_script("""
+        let [input] = arguments;
+        rect = input.getBoundingClientRect();
+        return rect["top"] >= 0 && rect["left"] >= 0 &&
+            (rect["top"] + rect["height"]) <= window.innerHeight &&
+            (rect["left"] + rect["width"]) <= window.innerWidth;
+            """, args=(element,)) is True
+
+
+@pytest.mark.parametrize("transform", ["translate(-100px, -100px)", "rotate(50deg)"])
+def test_element_not_interactable_css_transform(session, transform):
+    session.url = inline("""
+        <div style="width: 500px; height: 100px;
+            background-color: blue; transform: %s;">
+            <input type=button>                        
```

Comment:
There’s a couple of [linting failure](https://travis-ci.org/w3c/web-platform-tests/jobs/367312589) in CI due to lines like this, with whitespace at EOL.

Reply:
Mind moving interactability checks to a separate file like `interactability.py` as what we have for `element_send_keys`?

Reply:
fixed